Counter/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Counter</title>
+    <link rel="stylesheet" href="./style.css" />
```

Comment:
```suggestion
    <link rel="stylesheet" href="./style.css" />
    <script defer src="./js.js"></script>
```
Defer means that scripts will start to work only when the whole page is uploaded.